app/assets/stylesheets/pages/_home.scss

```diff
@@ -0,0 +1,197 @@
+.home {
+  @include clearfix();
+  @include breakpoint(min-width 900px) {
+    h2,
+    p {
+      text-align: left;
+      width: 100%;
+    }
+  }
+
+  background-color: $c-blue;
+  color: white;
+  min-height: 100vh;
+  overflow: hidden;
+  position: relative;
+}
+
+.home__header {
+  clear: both;
+  padding: rem(18) 0;
+}
+
+.home__logo {
+  @include fluid-size("font-size", 16px, 36px);
+  @include breakpoint(min-width 900px) {
+    margin: 0 0 0 rd(85, 1024);
+  }
+
+  color: white;
+  font: 600 $f-logo;
+  margin: 0 rd(20, 410);
+  text-decoration: none;
+  &:hover,
+  &:focus {
+    border-bottom: 4px solid white;
+    border-radius: 2px;
+  }
+}
+
+.home__inner {
+  @include breakpoint(min-width 1024px) {
+    padding: 5vh 0 0 0;
+  }
+
+  @include clearfix();
+  margin: 0 auto;
+  max-width: 2040px;
+  position: relative;
+}
+
+.home__inner-section {
+  @include breakpoint(min-width 1440px) {
+    max-width: 700px;
+  }
+
+  @include breakpoint(min-width 1024px) {
+    margin: 0;
+  }
+
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  margin: 0 auto;
+  max-width: 600px;
+}
+
+.home__svg-container {
+  @include breakpoint(min-width 1024px) {
+    margin: 10vh 0 0 0;
+    max-width: 1200px;
+    position: absolute;
+    right: 0;
+    width: 60%;
+  }
+
+  @include breakpoint(min-width 1200px) {
+    margin: calc(10vh - 5%) 0 0 0;
+  }
+
+  max-width: 700px;
+  position: relative;
+  svg {
+    position: relative;
+    right: -10%;
+    text-align: right;
+    top: 0;
+  }
+}
+
+.home__section {
+  @include breakpoint(min-width 1024px) {
+    float: left;
+    margin: 15vh 0 0 rd(85, 1024);
+    max-width: 700px;
+    width: 50%;
+  }
+
+  margin-top: rem(36);
+  position: relative;
+  z-index: 2;
+
+  h2 {
+    @include fluid-size("font-size", 20px, 36px);
+    @include breakpoint(min-width 1024px) {
+      max-width: none;
+      text-align: left;
+      width: 100%;
+    }
+
+    font-weight: 600;
+    line-height: 1.3;
+    margin: 0 auto rem(28) auto;
+    text-align: center;
+    width: rd(344, 410);
+  }
+}
+
+.home__nav {
+  @include breakpoint(min-width 1024px) {
+    padding-bottom: rem(135);
+    width: 95%;
+    order: 3;
+    justify-content: flex-start;
+  }
+
+  @include breakpoint(min-width 1440px) {
+    width: 88%;
+  }
+
+  @include breakpoint(400px 1023px) {
+    justify-content: space-around;
+  }
+
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 0 rem(34) 0;
+
+  a {
+    @include fluid-size("font-size", 14px, 16px);
+    @include breakpoint(min-width 1024px) {
+      margin: 0;
+      padding: rem(28) 0 rem(29) 0;
+      & + a {
+        margin-left: rd(60, 578);
+      }
+    }
+
+    background-color: $c-pink;
+    box-shadow: 0 9px 14px 0 rgba(0, 0, 0, .25);
+    color: white;
+    flex-basis: 1rem;
```

Comment:
Using `rem` for padding (which you've done in a few places) makes a liiiiittle more sense to me, since then the padding can match the font's height/line-height (and consistent spacing is generally visually pleasing).

Maybe think through where you're using `rem` a bit?  Try to make sure you're using `rem` when it's appropriate -- when sizes should be based off of the base font size.